docs/src/components/home/sections/CompatibleSection.tsx

```diff
@@ -0,0 +1,60 @@
+import { Flex, Heading, View } from '@aws-amplify/ui-react';
+import { SiTypescript } from 'react-icons/si';
+import { MdOutlineFlashOff, MdOutlineSupport } from 'react-icons/md';
+
+import { HomeCTA } from '@/components/home/HomeCTA';
+import { HomeFeatureCard } from '@/components/home/HomeFeatureCard';
+import { useIntersectionObserver } from '@/components/useIntersection';
+import { useRef } from 'react';
+
+export const CompatibleSection = ({ platform }) => {
+  const ref = useRef<HTMLDivElement | null>(null);
+  const entry = useIntersectionObserver(ref, {
+    threshold: 0.125,
+    freezeOnceVisible: true,
+  });
+  const isVisible = !!entry?.isIntersecting;
+
+  return (
+    <View
+      ref={ref}
+      as="section"
+      id="compatible"
+      className={`docs-home-section fade-in ${isVisible ? 'shown' : ''}`}
```

Comment:
We already take a dependency on [`classnames`](https://www.npmjs.com/package/classnames). We can use that to build the classname?

Reply:
Ah I keep forgetting about that!